store/store.go

```diff
@@ -192,8 +192,10 @@ func getStructFields(s interface{}) []string {
 	return fields
 }
 
+// Deltas enabled by default on classic, but allow opting in or out on both classic and core.
 func useDeltas() bool {
-	return osutil.GetenvBool("SNAPD_USE_DELTAS_EXPERIMENTAL")
+	deltasDefault := release.OnClassic
+	return osutil.GetenvBool("SNAPD_USE_DELTAS_EXPERIMENTAL", deltasDefault)
```

Comment:
Hi @zyga, that sounds like a great idea, but may be best approached by someone with more familiarity with snapd in a different branch given time constraints. Please do let me know if anything else is required for this branch to land, many thanks.